src/main/java/com/khutircraftubackend/userprofile/request/ChangeOfEmailRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.khutircraftubackend.userprofile.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+
+public record ChangeOfEmailRequest(
+
+        @NotBlank(message = "E-mail не може бути порожнім")
+        @Pattern(regexp = "^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}$")
+        String oldEmail,
+
+        @NotBlank(message = "E-mail не може бути порожнім")
+        @Pattern(regexp = "^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}$")
+        String newEmail
+) {
```

Comment:
we use this email validation in so many places,
can we consider having our own annotation for it ?
sort of :
```java
@ValidEmail
String email;
```